src/convertToObject.js

```diff
@@ -2,7 +2,8 @@
 
 /**
  * Implement convertToObject function:
- * Function gets string with styles
+ *
+ * Function takes string with styles
```

Comment:
Nit: each line should be almost 80 characters long.

Reply:
I agree with splitting based on meaning rather than just length

Reply:
https://github.com/google/guava/blob/master/guava/src/com/google/common/collect/AbstractIterator.java#L87 -- this is an example of good documentation. 

Different meaning separated into paragraphs. Lines within each paragraph are as close to 80 characters as possible.